warehouse/migrations/versions/a8ebe73ccaf2_make_githubpublisher_environment_non_.py

```diff
@@ -0,0 +1,73 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Make GitHubPublisher environment non-nullable
+
+Revision ID: a8ebe73ccaf2
+Revises: a2af745511e0
+Create Date: 2023-08-16 19:48:03.178852
+"""
+
+import sqlalchemy as sa
+
+from alembic import op
+
+revision = "a8ebe73ccaf2"
+down_revision = "a2af745511e0"
+
+
+def upgrade():
+    # Data migration
+    op.execute(
+        "UPDATE github_oidc_publishers SET environment = '' where environment = NULL"
```

Comment:
Use `IS NULL` instead of equality.

> Do not write expression = NULL because NULL is not “equal to” NULL. (The null value represents an unknown value, and it is not known whether two unknown values are equal.)

https://www.postgresql.org/docs/14/functions-comparison.html


```sql
warehouse=> select count(*) from pending_github_oidc_publishers where environment = NULL;
 count
-------
     0

warehouse=> select count(*) from pending_github_oidc_publishers where environment IS NULL;
 count
-------
   873
```

Reply:
fool me once, shame on me, fool me twice, shame on... shame on nulls.